omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -112,6 +115,17 @@ public List<Role> getRoles(WebRequest request) {
 	public String showForm(@RequestParam(required = false, value = "userId") Integer userId,
 	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson,
 	        @ModelAttribute("user") User user, ModelMap model) {
+
+		try {
+			boolean isPlatform22OrNewer = isPlatform22OrNewer();
```

Comment:
Do we need the above variable? Why not just directly call the method in the if statement?